specs/binding-grpc-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/grpc/kafka/schema/grpc.kafka.schema.patch.json

```diff
@@ -233,7 +240,8 @@
                                                 {
                                                     "title": "Acks",
                                                     "type": "string",
-                                                    "enum": [ "none", "leader_only", "in_sync_replicas" ]
+                                                    "enum": [ "none", "leader_only", "in_sync_replicas" ],
+                                                    "default": "in_sync_replicas"
```

Comment:
The added defaults were communicated in the documentation but not in the schema.